src/editor/UI/Minimap.re

```diff
@@ -62,6 +62,10 @@ let component = React.component("Minimap");
 let absoluteStyle =
   Style.[position(`Absolute), top(0), bottom(0), left(0), right(0)];
 
+let getMinimapSize = (view: Editor.t) =>
+  view.viewLines < Editor.getVisibleView(view)
```

Comment:
is it worth saving the value of `Editor.getVisibleView(view)` in a variable since either side of this ternary means it gets called, but on one side it gets called twice :man_shrugging: 	all in all a very minor issue